commons/Cargo.toml

```diff
@@ -28,7 +28,7 @@ thiserror = "1"
 walkdir = "2"
 ascii_table = { version = "4", features = ["color_codes"] }
 const_format = "0.2"
-fun_run = "0.1"
+fun_run = "0.1.1"
```

Comment:
The existing style used in this repo (and across all of our CNB repos) is to only put the major version in `Cargo.toml` and leave version resolution within that major version range to the lockfile.

Reply:
@schneems I've triggered a Dependabot run manually now - which opened #234, that contains the `fun_run` update (amongst a couple of small other updates).

One thing I noticed that was the `fun_run` crate is missing some metadata (such as the GitHub repo URL) - which is why the changelog didn't show up on the Dependabot PR, and why there is no GitHub repo link on the crates.io page:
https://crates.io/crates/fun_run

For an example of the metadata you might want to include, see libcnb's manifest:
https://github.com/heroku/libcnb.rs/blob/5ab4b036a8dd18dc2d25a3d3ee49ed66ef3ec9d3/libcnb/Cargo.toml#L7-L12

The upstream docs have some more details on each of those fields:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-repository-field